<ui version="4.0" >
 <class>ConnectionEditor</class>
 <widget class="QWidget" name="ConnectionEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>287</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabWired" >
      <attribute name="title" >
       <string>Wired</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QTreeWidget" name="listWired" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QPushButton" name="addWired" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editWired" >
           <property name="text" >
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteWired" >
           <property name="text" >
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWireless" >
      <attribute name="title" >
       <string>Wireless</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QTreeWidget" name="listWireless" >
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <item>
          <widget class="QPushButton" name="addWireless" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editWireless" >
           <property name="text" >
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteWireless" >
           <property name="text" >
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCellular" >
      <attribute name="title" >
       <string>Mobile Broadband</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <widget class="QTreeWidget" name="listCellular" >
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <item>
          <widget class="QPushButton" name="addCellular" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editCellular" >
           <property name="text" >
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteCellular" >
           <property name="text" >
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVpn" >
      <attribute name="title" >
       <string>VPN</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4" >
       <item>
        <widget class="QTreeWidget" name="listVpn" >
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" >
         <item>
          <widget class="QPushButton" name="addVpn" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editVpn" >
           <property name="text" >
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteVpn" >
           <property name="text" >
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPppoe" >
      <attribute name="title" >
       <string>DSL</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5" >
       <item>
        <widget class="QTreeWidget" name="listPppoe" >
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Connection</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last used</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" >
         <item>
          <widget class="QPushButton" name="addPppoe" >
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editPppoe" >
           <property name="text" >
            <string>&amp;Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePppoe" >
           <property name="text" >
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
